internal/support/notifications/sending_service.go

```diff
@@ -141,3 +143,70 @@ func handleFailedTransmission(t models.Transmission) {
 		}
 	}
 }
+
+// validateLine checks to see if a line has CR or LF as per RFC 5321
+func validateLine(line string) error {
```

Comment:
Yes, it makes sense to have this check at entry point so that if the mistake occurs it can be reported back right away, and can be correct at the source, but OTOH it serves well as a secondary check, will take a look at it. Thanks!

Reply:
I have moved this check up to the source, that is, to subscribe api, subscriptionHandler. If a CRLF pair is found the api responds back with the message that email addresses contains CRLF, and returns. It is in both PUT and POST methods. The function that implements it (validateEmail) probably can be enhanced further later to add more email validation. I have not been able to get to the check for addresses read from .toml file, code is more complicated there. I am inclining to handle that as a separate PR.